### ISSUE TITLE
esy fails on freshly cloned repo (when building OCaml?)

### ISSUE THREAD

**Issue by DLFW** [esy-ocaml/hello-reason#75]
I cloned the project, and started `esy`, which fails with
```
...
    gcc -O2 -fno-strict-aliasing -fwrapv -Wall -fno-tree-vrp -g -D_FILE_OFFSET_BITS=64 -D_REENTRANT -DCAML_NAME_SPACE  -DOCAML_STDLIB_DIR='"/home/daniel/.esy/3___________________________________________________________________/s/ocaml-4.8.1000-e74c5c0d/lib/ocaml"'  -Wl,-E -o ocamlruni prims.o libcamlruni.a -lm -ldl  -lpthread 
    /usr/bin/ld: libcamlruni.a(backtrace_bi.o):/home/daniel/.esy/3/b/ocaml-4.8.1000-e74c5c0d/runtime/backtrace.c:31: multiple definition of `caml_debug_info'; /usr/bin/ld: libcamlruni.a(backtrace_byt_bi.o):/home/daniel/.esy/3/b/ocaml-4.8.1000-e74c5c0d/runtime/backtrace_byt.c:47: first defined here
    libcamlrund.a(backtrace_bd.o):/home/daniel/.esy/3/b/ocaml-4.8.1000-e74c5c0d/runtime/backtrace.c:31: multiple definition of `caml_debug_info'; libcamlrund.a(backtrace_byt_bd.o):/home/daniel/.esy/3/b/ocaml-4.8.1000-e74c5c0d/runtime/backtrace_byt.c:47: first defined here
    collect2: error: ld returned 1 exit status
    make[1]: *** [Makefile:301: ocamlruni] Error 1
    make[1]: *** Waiting for unfinished jobs....
    collect2: error: ld returned 1 exit status
    make[1]: *** [Makefile:295: ocamlrund] Error 1
    make[1]: Leaving directory '/home/daniel/.esy/3/b/ocaml-4.8.1000-e74c5c0d/runtime'
    make: *** [Makefile:323: coldstart] Error 2
    error: command failed: './esy-build' (exited with 2)
    esy-build-package: exiting with errors above...
    
  building ocaml@4.8.1000
esy: exiting due to errors above
```
- I’m on Arch Linux
- Esy version: 0.6.6

**Comment by JohnPucciarelli** [comment]
Running into the same issue. esy 0.6.6 as well. Fedora linux.

Edit: Issue seems to be related to **gcc 10**.

I was able to workaround the issue by following the advice here:
https://github.com/esy/esy/issues/1126

**TL;DR;**
Create a wrapper for gcc at `/usr/local/bin/gcc`:
```
#! /bin/sh

eval /usr/bin/gcc -fcommon '"$@"'
```
https://discuss.ocaml.org/t/opam-switch-create-and-gcc10/5864/13

**Comment by DLFW** [comment]
Thanks!
Works! (around... :) )

Not sure if this hello-reason project needs to fix some dependency spec (OCaml version) once the OCaml people have fixed that? Should this issue remain open? (Sorry, Reason/OCaml noob here..)

**Comment by JohnPucciarelli** [comment]
Glad it worked for you. I'll leave it to you, whether you want to keep this open or close it.

Probably makes sense to file an issue or comment on the one I linked to in the esy repository.
